components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/client/model/AuthenticationRequestWrapper.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
```

Comment:
Inc. --> LLC
Check the template

Reply:
Resolved in [1f7f018](https://github.com/wso2/carbon-identity-framework/pull/4176/commits/1f7f01855ee1e850acaf931e8daa999a49bc276c)